designs/credentialManagement/images/classDiagram.svg

```diff
@@ -64,61 +64,13 @@ ToolkitCredentialsProvider "many" o- - "1" CredentialProviderFactory: Creates
 CredentialProviderFactory "many" o- - "1" CredentialManager: Owns
 @enduml
 
-@startuml
-top to bottom direction
-
-class AwsRegion {
```

Comment:
where did all this stuff go?

Reply:
It was duplicated, it's actually in here twice